script/entrypoint.sh

```diff
@@ -13,10 +13,12 @@ TRY_LOOP="20"
 : "${POSTGRES_DB:="airflow"}"
 
 # Defaults and back-compat
+: "${AIRFLOW__HOME:="/usr/local/airflow"}"
```

Comment:
Possible Typo? 

I guess this has to be `AIRFLOW_HOME`, with a single underscore (`_'), instead of `AIRFLOW__HOME` with double underscores (`_ _`) .

When I start airflow using changes in this PR airflow uses the location `~/airflow` which evaluates to `/usr/local/airflow/airflow/`.

I am suspecting this line to be the cause, and fixing it should solve the problem.

Reply:
And `AIRFLOW_HOME` is already set https://github.com/puckel/docker-airflow/blob/024cf2d08d2de1edf63f780c4d60c5d8fc70b265/Dockerfile#L16

Reply:
Just pushed change to fix that.

Reply:
> And `AIRFLOW_HOME` is already set
> 
> https://github.com/puckel/docker-airflow/blob/024cf2d08d2de1edf63f780c4d60c5d8fc70b265/Dockerfile#L16

As far as I remember the `AIRFLOW_HOME` in `Dockerfile` means the Home Directory of the Container's user. So it overrides the user like `/home/alvyl/` with `/usr/local/airflow` or aliased by `~` (tilde).

So by default Airflow uses `~/airflow`. And it turns out to be `/home/alvyl/airflow` or  in this case its `/usr/local/airflow/airflow` which is wrong.

The `AIRFLOW_HOME` in the `entrypoint.sh` is the actual airflow path that overrides `~/airflow`.

Hope this makes sense.